classes/Pods.php

```diff
@@ -1696,7 +1697,7 @@ public function field( $name, $single = null, $raw = false ) {
 			$value = array_merge( ...$value );
 		}
 
-		if ( ! empty( $field_data ) && ( $params->display || ! $params->raw ) && ! $params->in_form && ! $params->raw_display ) {
+		if ( ! $is_mapped && ! empty( $field_data ) && ( $params->display || ! $params->raw ) && ! $params->in_form && ! $params->raw_display ) {
```

Comment:
In all cases, are mapped values handling their own display output? Maybe, just worth confirming

Reply:
I did assume they would. Most mappers are related to iteration fields (index, row etc.).
There are two mappers that our unit tests should verify: Avatar and Image fields.

That said, I feel like mappers are and should be short circuit handlers for fields and thus should not have to be processed further.
The `Pods::field()` method should definitely be more abstracted to make this much more easy to follow and debug.